app/src/main/java/com/example/bigowlapp/repository/ViewUserRepository.java

```diff
@@ -0,0 +1,32 @@
+package com.example.bigowlapp.repository;
```

Comment:
The only class that uses this is `ViewUserViewModel.java`, which is unused, so I'm guessing it should be removed.

It also has code related to the Firebase realtime database, which we are not using.

Reply:
It has been removed

Reply:
I think its still in the repo.

Reply:
ok done